ginivision/src/main/java/net/gini/android/vision/review/multipage/MultiPageReviewFragment.java

```diff
@@ -397,6 +397,10 @@ public void onStart() {
         if (!mPreviewsShown) {
             observeViewTree();
         }
+        uploadDocuments();
```

Comment:
Why don't you upload the documents when they are created instead of when multipage fragment starts?

Reply:
To be in line with the single-page review screen I want to do uploads once the user is in the review screen.

I also want to keep upload out of the camera screen to not have to deal with passing upload related information (error, progress) to the review screen.